app/src/main/java/de/danoeh/antennapod/fragment/preferences/NetworkPreferencesFragment.java

```diff
@@ -174,4 +223,162 @@ private void showUpdateIntervalTimePreferencesDialog() {
         }
         return entries;
     }
+
+
+    public class ViewDialog {
+
+
+        final Context context = getActivity();
+        int string_spinnercase;
+
+        public void showDialog(Activity activity, String msg){
+            final Dialog dialog = new Dialog(activity);
+            dialog.requestWindowFeature(Window.FEATURE_NO_TITLE);
+            dialog.setCancelable(false);
+            dialog.setContentView(R.layout.update_refresh_time_dialog);
+            Spinner spinner = (Spinner) dialog.findViewById(R.id.spinner);
+
+            RadioButton radioButton_interval = (RadioButton) dialog.findViewById(R.id.intervalRadioButton);
+            RadioButton radioButtonTime = (RadioButton) dialog.findViewById(R.id.timeRadioButton);
+            RadioButton radioButton_disable = (RadioButton) dialog.findViewById(R.id.disableButton);
+            RadioGroup radioGroupOne = (RadioGroup) dialog.findViewById(R.id.genderRadioGroup);
+            TimePicker timePicker = (TimePicker) dialog.findViewById(R.id.timePicker);
+            Button button_accept = (Button) dialog.findViewById(R.id.button_accept);
+            Button button_cancel = (Button) dialog.findViewById(R.id.button_cancel);
+
+
+
+            radioGroupOne.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener() {
+                @Override
+                public void onCheckedChanged(RadioGroup radioGroup, int i)
+                {
+                    switch (i)
+
+                    {
```

Comment:
Please use the Google code style. You can import the configuration into Android studio or just use the CI errors to change the style.